ipyleaflet/leaflet.py

```diff
@@ -1355,7 +1359,8 @@ def _default_colormap(self):
     def _default_style_callback(self):
         def compute_style(feature, colormap, choro_data):
             return dict(
-                fillColor=colormap(choro_data),
+                fillColor = self.nan_color if math.isnan(choro_data) else colormap(choro_data),
+                fillOpacity = self.nan_opacity if math.isnan(choro_data) else 0.8,
```

Comment:
The linter is complaining about the extra spaces:

```suggestion
                fillColor=self.nan_color if math.isnan(choro_data) else colormap(choro_data),
                fillOpacity=self.nan_opacity if math.isnan(choro_data) else 0.8,
```